File: frontend/travel-with-anna-ui/src/app/services/models/day-response.ts
/* tslint:disable */
/* eslint-disable */
import { ActivityListResponse } from '../models/activity-list-response';
import { NoteResponse } from '../models/note-response';
export interface DayResponse {
  activities?: ActivityListResponse;
  date?: string;
  dayId?: number;
  dayNumber?: number;
  dayOfWeek?: string;
  isToday?: boolean;
  note?: NoteResponse;
  tripId?: number;
}

File: backend/travel-with-anna/src/main/java/com/andwis/travel_with_anna/trip/day/DayResponse.java
package com.andwis.travel_with_anna.trip.day;

import com.andwis.travel_with_anna.trip.day.activity.ActivityListResponse;
import com.andwis.travel_with_anna.trip.note.NoteResponse;

import java.time.LocalDate;

public record DayResponse(
        Long dayId,
        LocalDate date,
        String dayOfWeek,
        boolean isToday,
        int dayNumber,
        NoteResponse note,
        Long tripId,
        ActivityListResponse activities
) {
}
